tests/s-thread-pthread_exit.c

```diff
@@ -0,0 +1,27 @@
+#include <pthread.h>
+
```

Comment:
I think you need to add `<stdio.h>` as well.  Also I prefer a short file name like `s-thread-exit.c`.